utils/mvn/utils.go

```diff
@@ -113,6 +120,11 @@ func runMvnVersionCommand(mavenHome string) ([]string, error) {
 }
 
 func parseMvnHome(line string) (string, error) {
+	// Line example: 'Maven home: /usr/share/maven'
+	
+	// Remove all prefix before 'Maven' (if exists)
+	line = line[strings.Index(line,"Maven"):]
```

Comment:
Please run `go fmt ./...`
```suggestion
	line = line[strings.Index(line, "Maven"):]
```